codec/testutil/codec.go

```diff
@@ -19,20 +19,13 @@ type CodecOptions struct {
 // NewInterfaceRegistry returns a new InterfaceRegistry with the given options.
 func (o CodecOptions) NewInterfaceRegistry() codectypes.InterfaceRegistry {
```

Comment:
The type `CodecOptions` defined in line 14 above has options for "normal" account addresses and validator accounts, which configure the address prefixes.
Similar to a previous comment, do those prefixes make sense in Hex addresses?

Reply:
Correct, no usage whatsoever in `hex` addresses. Kept for compatibility (and using empty strings for such values).
So, if I'll delete the `HexPrefix`, I'll also make sure to remove such `CodecOptions`

Reply:
Removed in latest commit

Reply:
Resolved